algoliasearch-core/src/test/java/com/algolia/search/JacksonParserTest.java

```diff
@@ -171,18 +171,18 @@ void serializeFacetFilters() throws IOException {
   @ValueSource(strings = {"facetFilters", "optionalFilters", "tagFilters", "numericFilters"})
   void testLegacyFiltersFormat(String input) throws IOException {
```

Comment:
We also need a test for that legacy use-case:

> (color:green, color:yellow) is equivalent to [["color:green", "color:yellow"]]

Reply:
True! I've added the support for this case too! :D